pkg/installmanager/installmanager.go

```diff
@@ -1465,7 +1466,7 @@ func uploadAdminKubeconfig(m *InstallManager) (*corev1.Secret, error) {
 	kubeconfigSecret.Labels = k8slabels.AddLabel(kubeconfigSecret.Labels, constants.ClusterProvisionNameLabel, m.ClusterProvision.Name)
 	kubeconfigSecret.Labels = k8slabels.AddLabel(kubeconfigSecret.Labels, constants.SecretTypeLabel, constants.SecretTypeKubeConfig)
 
-	provisionGVK, err := apiutil.GVKForObject(m.ClusterProvision, scheme.Scheme)
+	provisionGVK, err := apiutil.GVKForObject(m.ClusterProvision, testfake.GetScheme())
```

Comment:
Sometimes you're inlining `testfake.GetScheme()` and sometimes you're creating a local var. I don't super care which, just thought you might prefer to be consistent.